.github/workflows/release.yml

```diff
@@ -4,34 +4,44 @@ on:
   release:
     types: [created]
 
+env:
+  GORELEASER_VERSION: v1.24.0
+  GO_VERSION: 1.21.1
+
+permissions:
+  contents: write
+
 jobs:
   go-test:
     name: Test
     uses: ./.github/workflows/test.yml
 
-  go-build:
-    name: Build
-    uses: ./.github/workflows/build.yml
-    with:
-      create_release: true
-    secrets: inherit
-
-  release:
+  goreleaser:
     name: Release World CLI binary
-    runs-on: ubuntu-latest
-    needs: [go-test, go-build]
-    strategy:
-      matrix:
-        go-version: [1.21.x]
+    runs-on: namespace-profile-linux-4vcpu-8gb-cached
```

Comment:
Don't think we need a custom runner for this. Would rather not use a custom runner if we don't need to because we have limited concurrency.